content/en/docs/concepts/services-networking/service.md

```diff
@@ -167,6 +167,9 @@ subsets:
       - port: 9376
 ```
 
+The name of the Endpoints object must be a valid
```

Comment:
Hello @tengqm . Generally, the changes look good.
I questioned why Endpoints is singled out? What about Service and EndpointSlices?

Reply:
OK. I see that you updated these resources in another PR.